src/components/GlobeOptimized.jsx

```diff
@@ -656,23 +656,22 @@ export default function GlobeOptimized(props) {
     //   globeAttributes.current.land = land;
     //   render();
     // });
-
     // register drag callbacks
     const canvas = select('#globe');
-
     canvas.call(
-      // drag().on('start', dragstarted).on('drag', dragged).on('end', dragended)
       drag().on('drag', dragged)
     );
     
-
     canvas.on('click', clicked);
-
     canvas.on('mousemove', havoured);
 
+    const user_seti_len = Cookies.get('user');
```

Comment:
`user` is probably not a good idea as the cookie name. Can we use any other name that's more descriptive?

Reply:
updated 'user' to 'seti_cams_user_2022'